### PR TITLE
Fix / logs in inertia

### LINKED ISSUE #35: Admin - Logs

### PR DESCRIPTION
**Pull request by SedlarDavid**
Closes #35

### CODE REVIEW

File: resources/js/Components/Modals/LogsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { Utils } from "@/Tools/Utils";
+import { ITopic } from "@/Components/Modals/TopicModal";
+
+export interface ILog {
+    message: string;
+    context: string;
+}
+
+interface ILogModal {
+    onClose: () => any;
```
Comment (tomesfilip) [inline resources/js/Components/Modals/LogsModal.tsx:25]: this type could be `() => void;`

File: resources/js/Components/Modals/LogsModal.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    const [errors, setErrors] = useState<string | null>(null);
+    const [data, setData] = useState<ILog>(
+        log ?? {
+            message: "",
+            context: "",
+        },
+    );
+
+    return (
+        <Modal show={isVisible} onClose={onClose} size={"l"}>
```
Comment (tomesfilip) [inline resources/js/Components/Modals/LogsModal.tsx:40]: if using string values, no need to use curly brackets `size="1"`

File: resources/js/Pages/Admin/Logs.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        isVisible: boolean;
+        log: ILog | null;
+    }>({
+        isVisible: false,
+        log: null,
+    });
+
+    function onLogDetail(log: any) {
+        if (log.level_name === "NOTICE") {
+            console.log(log);
```
Comment (tomesfilip) [inline resources/js/Pages/Admin/Logs.tsx:20]: do we want to keep this console log or is it a leftover?

**Review by tomesfilip — APPROVED** [review]
couple improvements, otherwise GJ

File: resources/js/Pages/Admin/Logs.tsx (hunk 3 above)
Comment (SedlarDavid) [reply resources/js/Pages/Admin/Logs.tsx:20]: leftover, thanks :)

**Pull request merged**